docs/source/standard.md

```diff
@@ -0,0 +1,147 @@
+(standard)=
+
+# A Standard Architecture for TREs
+
+<!-- What this document intends to do (and what it doesn't), the level of detail we aim for contrasted with other technical standards -->
+
+The SATRE standard has been developed based on the following principles:
+
+- TREs should be as as easy as possible for end-users to use (e.g. researchers) whilst still remaining secure.
+- TRE deployments should be offered that support data of different levels of sensitivity (e.g. through a tiered system of technical controls and policies).
+- TREs conforming to the standard should be interoperable and provide a familiar end-user experience.
+- The standard will be managed and updated following an open, community-driven process, and will not be tied to a single vendor or implementation.
+
+The SATRE standard is based around the idea of capabilities, some of which are required and some of which are optional.
+Any particular TRE implementation should be able to score itself against each capability as either "supported", "partially supported" or "unsupported" (see {ref}`evaluation` for details). The capabilities are grouped into the following broad categories:
+
+- information governance
+- computing technology
+- data management
+- information security
+
+In addition to these capabilities, any organisation running a TRE will need to consider:
```

Comment:
I think we'll need to be clearer about the different audiences for this spec. Capabilities mostly apply to a TRE implementation/code-base, whereas governance and other requirements on organisations apply to the TRE deployer. Are we saying it's the implementation, the deployment, or both that "comply" with the standard?

Reply:
I think it can only really be a deployment that complies. If it's just an implementation then we're sidestepping the (important) questions of governance.

Reply:
> TRE deployments should be offered that support data of different levels of sensitivity (_e.g._ through a tiered system of technical controls and policies).

Something like this is a requirement on the implementation though, the deployment will only be at a single tier.

Reply:
Since there's a separate discussion over whether or not to include tiers it's worth pointing out the implementation vs deployments is relevant in other areas, e.g. ["allow pasting in to be disabled"](https://github.com/sa-tre/satre-specification/issues/53#issuecomment-1558957657) is a capability of the implementation, but if the standard is for the deployment then we can only state the minimum requirement which is to forbid copying out.

Reply:
I've created a new issue https://github.com/sa-tre/satre-specification/issues/74
but I think we need to resolve it ASAP